v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/transforms/WriteToElasticsearch.java

```diff
@@ -0,0 +1,155 @@
+package com.google.cloud.teleport.v2.elasticsearch.transforms;
```

Comment:
Please move to `com.google.cloud.teleport.v2.transforms` package (no "elasticsearch" in the package name).